batcher/aligned-batcher/src/lib.rs

```diff
@@ -1559,6 +1560,11 @@ impl Batcher {
             ETHEREUM_CALL_MAX_RETRY_DELAY,
         )
         .await;
+        self.metrics
+            .create_new_task_latency
+            .set(start.elapsed().as_millis() as i64);
```

Comment:
The metric should be per attempt I think. We should also count the retries.

Reply:
Measurement of the hole function, including retries, was the request. Maybe we should discuss this again.

Reply:
Maybe we should rename the metric name to "Duration" instead of latency